Source/common/SNTConfigurator.h

```diff
@@ -151,9 +151,10 @@
 
 ///
 ///  Defines how event logs are stored. Options are:
-///    SNTEventLogTypeSyslog: Sent to ASL or ULS (if built with the 10.12 SDK or later).
-///    SNTEventLogTypeFilelog: Sent to a file on disk. Use eventLogPath to specify a path.
-///    SNTEventLogTypeProtobuf: (BETA) Sent to a file on disk, using maildir format. Use
+///    SNTEventLogTypeSyslog "syslog": Sent to ASL or ULS (if built with the 10.12 SDK or later).
+///    SNTEventLogTypeFilelog "": Sent to a file on disk. Use eventLogPath to specify a path.
```

Comment:
The empty string is a bit confusing. Maybe add a note that  `SNTEventLogTypeFilelog` is the default option and doesn't have an associated value in the config. Or maybe we should make a value folks can use, just to be explicit?

Reply:
Made "file" an option